py3status/modules/rss_aggregator.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+"""
+Display the unread feed items in your OwnCloud/NextCloud account.
+
+You can also decide to check only for specific feeds or folders of feeds. To use this
+feature, you have to first get the IDs of those feeds or folders. You can get those IDs
+by clicking on the desired feed or folder and watching the URL:
+```
+https://yourcloudinstance.com/index.php/apps/news/#/items/feeds/FEED_ID
+https://yourcloudinstance.com/index.php/apps/news/#/items/folders/FOLDER_ID
+
+```
+If both feeds list and folders list are left empty, all unread feed items will be counted.
+Counted items don't need to match both feeds and folders lists. Items matching both a
+given feed ID and a given folder ID will be counted only once.
+
+Configuration parameters:
+    aggregator: feed aggregator used, see note below. For now, the only supported value is
+        `'owncloud'`, that means that you'll only be able to configure a
+        OwnCloud/NextCloud instance. Other feed aggregator may be supported in future
+        releases. (default 'owncloud')
+    cache_timeout: how often to run this check (default 60)
+    feed_ids: list of IDs of feeds to watch, see note below (default [])
+    folder_ids: list of IDs of folders ro watch (default [])
+    format: format to display (default 'Feed: {unseen}')
+    password: login password (default '<PASSWORD>')
+    server: aggregator server to connect to (default 'https://yourcloudinstance.com')
+    user: login user (default '<USERNAME>')
+
+Format placeholders:
+    {unseen} sum of numbers of unread feed elements
+
+Color options:
+    color_new_items: text color when there is new items (default color_good)
+    color_error: text color when there is an error (default color_bad)
+
+Requires:
+    python-requests: library to make web requests
+
+@author raspbeguy
+"""
+
+import requests
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
+    aggregator = 'owncloud'
+    cache_timeout = 60
+    feed_ids = []
+    folder_ids = []
+    format = 'Feed: {unseen}'
+    password = '<PASSWORD>'
+    server = 'https://yourcloudinstance.com'
+    user = '<USERNAME>'
```

Comment:
I'd be inclined to have this and password default to None so you can tell if a user actually set it or not.   Also if not set then in the request just do `auth=None`